### ISSUE TITLE
Highlighted county is just default Leaflet styling

### ISSUE THREAD

**Issue by almccon** [americanpanorama/panorama-forcedmigration#11]
We should actually think about that ugly blue highlight, and tweak it. 
![screen shot 2016-02-26 at 3 11 43 pm](https://cloud.githubusercontent.com/assets/1212178/13368050/15e06422-dc9c-11e5-9cad-247ac30630e2.png)

**Comment by ebrelsford** [comment]
I didn't see this issue at the time, but when working on #21 I found some CSS that was attempting to style the selected county and fixed it. It's a little hard to see so we probably still want to think about how to highlight counties?

![selection_001](https://cloud.githubusercontent.com/assets/375002/13884780/dbc9b354-ed05-11e5-9b1d-553e2810265b.png)

**Comment by almccon** [comment]
Agreed, probably still needs some design thinking. But I'm glad it's no longer just the default!

@nicolettehayes when you get a chance to do a batch of design work about Panorama, give this some thought please!

**Comment by rnelson2** [comment]
I'm experimenting with conveying more info for counties when a state is selected using a choropleth for relative number of migrations. I like this. What do you think? 

Selected counties still could use some better styling, if nothing else bringing the county to the top to even  the stroke.

![screen shot 2016-03-24 at 3 34 59 pm](https://cloud.githubusercontent.com/assets/8862699/14029123/ae7b9338-f1d6-11e5-9281-ddf219e685cb.png)
